docs/automated-process.md

```diff
@@ -0,0 +1,32 @@
+# How to release a new version of the Ansible Community Package - Automated Release Process
+
+## Preamble
+
+This document describes the (mostly) automated ansible community package release process.
+The automated processes uses GitHub Actions to automate the [manual release process](release-process.md).
+
+!!! note
+    Throughout this page, placeholder values in code blocks are formatted as
+    `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to specify.
+
+
+## Trigger the workflow
+
+1. Trigger [the automated workflow](https://github.com/ansible-community/ansible-build-data/actions/workflows/ansible-release.yml) on the **Actions** tab of the repository and specify the release version, such as 11.2.0, and the major version, such as 11.
+   The process will create a PR in the [`ansible-build-data` repository](https://github.com/ansible-community/ansible-build-data/).
+   Afterwards it will wait for approval before continuing with uploading the package to PyPI.
+   All users with commit rights in the `ansible-build-data` repository will be informed with a notification once the approval is needed.
```

Comment:
I believe only @ansible-community/release-management-wg is notified.

Reply:
Why do you think so? That group isn't mentioned in the workflow, so I would assume that GH notifies everyone with appropriate rights, which probably is the set of folks with commit rights.

Reply:
I believe that's how the environment that @anweshadas created is set up. It's set in the repo settings, not the workflow file.

Reply:
@anweshadas can you confirm this?

Reply:
> Why do you think so? That group isn't mentioned in the workflow, so I would assume that GH notifies everyone with appropriate rights, which probably is the set of folks with commit rights.

In GitHub environments, there's protection rules, of which one is "required reviewers". These are listed as GitHub usernames or teams. The maximum is 6 entries. Whoever is listed will be notified, I don't think this will notify accounts that aren't in those teams or lists.

Reply:
I don't have any way to verify this (I don't have access to the settings), so can someone with appropriate access please check this and confirm (or say what else it is)?

Reply:
I don't have enough privileges in this repo to check this :(

Maybe @oraNod does?

Reply:
Hi, plz excuse the delayed response. Just getting back from PTO...

It looks like @gotmax23 is correct and only folks in the `ansible-community/release-management-wg` group are notified. Here's a screenshot of the required reviewers section of the environment:

![image](https://github.com/ansible-community/ansible-build-data/assets/32749632/28e272fa-47f0-42f9-bae7-d4ef125a2372)

On a side note, I'm surprised that @felixfontein and @webknjaz don't have sufficient repo permissions. Should we address that?

Reply:
Thanks a lot! I've updated the paragraph in 81cc67840cdc38bf8b9e95557961b727c190dce4 and added a footnote explaining how this is configured.

Regarding repo permissions: I wouldn't mind having more permissions :) I have no idea who currently has more permissions and why the permissions are as they are...